Tests/SwiftLintFrameworkTests/RuleConfigurationTests.swift

```diff
@@ -10,8 +10,127 @@ import SourceKittenFramework
 @testable import SwiftLintFramework
 import XCTest
 
+// swiftlint:disable file_length
```

Comment:
The number of tests this adds as well as the silenced SwiftLint length violations should be an indicator that it might be time to move line length rule and configuration tests into their own test file. Could you please do that? Also please remember to update `LinuxMain.swift`.